<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Generate3DGridDialog</class>
 <widget class="QDialog" name="Generate3DGridDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate 3D Grid</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QDialog {
        background-color: #2E2E2E;
        color: #FFFFFF;
    }
    QLineEdit {
        background-color: #3C3C3C;
        color: #FFFFFF;
        border: 1px solid #555555;
        padding: 5px;
    }
    QLabel {
        color: #FFFFFF;
    }
    QPushButton {
        background-color: #4A4A4A;
        color: #FFFFFF;
        border: 1px solid #555555;
        padding: 5px;
    }
    QPushButton:hover {
        background-color: #5A5A5A;
    }
    QPushButton:pressed {
        background-color: #3A3A3A;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelRows">
     <property name="text">
      <string>Rows:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditRows">
     <property name="text">
      <string>10</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelColumns">
     <property name="text">
      <string>Columns:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditColumns">
     <property name="text">
      <string>10</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelLayers">
     <property name="text">
      <string>Layers:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditLayers">
     <property name="text">
      <string>10</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelCellSize">
     <property name="text">
      <string>Cell Size:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditCellSize">
     <property name="text">
      <string>1.0</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>Generate3DGridDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>Generate3DGridDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>